### PR TITLE
[Module][WIP] Antsbet

### PR DESCRIPTION
**Pull request by ThoumyreStanislas**
add module bet t1 with antsbrainextraction.sh
test passed correctly

### CODE REVIEW

**Comment by ThoumyreStanislas** [comment]
The tests failed because I added new data and tried with a local direction for the data and the test passed. I then changed the directory to nf-scil and the test failed because the data was missing from nf-scil.

**Comment by AlexVCaron** [comment]
Can you share me the missing test data ? Either in an archive or directly here as links. I'll add them so they are available for your test case using the Scilpy_Fetcher

**Comment by AlexVCaron** [comment]
The test package is online, the name : `antsbet.zip` !

File: modules/nf-scil/betcrop/antsbet/main.nf [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+process BETCROP_ANTSBET {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+
```
Comment (AlexVCaron) [inline modules/nf-scil/betcrop/antsbet/main.nf:11]: remove newline

File: modules/nf-scil/betcrop/antsbet/main.nf [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    tuple val(meta), path(t1), path(template), path(map)
```
Comment (AlexVCaron) [inline modules/nf-scil/betcrop/antsbet/main.nf:12]: Rename `map` to `tissues_probabilities` or something more telling

File: modules/nf-scil/betcrop/antsbet/meta.yml
Comment (AlexVCaron) [inline modules/nf-scil/betcrop/antsbet/meta.yml:1]: Capitalize `T1` in descriptions this file (don't capitalize variables and symbols)

**Review by AlexVCaron — CHANGES_REQUESTED** [review]
Some really minor comments. I'll review further once tests are running :wink:

**Comment by AlexVCaron** [comment]
The archive is up-to-date on the main branch ! I placed two variations of the `T1`image :

- `t1.nii.gz` which is really close to the template (not the best test case)
- `t1_unaligned.nii.gz` which I virtually moved away further (two rotations and a translation, better as a test)

Can you try recreating the test yml files with the `unaligned` image, see if it passes ?

**Review by AlexVCaron — APPROVED** [review]

**Pull request merged**